src/components/molecules/conceptLink/ConceptLink.tsx

```diff
@@ -32,7 +34,11 @@ export const ConceptLink = ({ concept, onClick, triggerClasses = "" }: IProps) =
     <span className={allTriggerClasses}>{title}</span>
   );
 
-  return (
+  return children ? (
+    <Popover openOnHover onOpenChange={setIsOpen} trigger={trigger}>
+      {children}
```

Comment:
Critique: The original intent of `ConceptLink` was to provide an opinionated way to render information about a topic. Both of the changes made here are hard overrides which feels like a circumvention of that intent.

Suggestions:
- Rather than having a `label` prop override, add a `count` prop to append the count to the title.
- Rather than replacing the entire popover content via `children`, add a `callToAction` ReactNode prop which defaults to the source link (original behaviour) but can be replaced by the list of documents.